django/utils/log.py

```diff
@@ -160,9 +160,11 @@ def filter(self, record):
 
 
 class ServerFormatter(logging.Formatter):
-    def __init__(self, *args, **kwargs):
+    def __init__(self, fmt=None, datefmt=None, style='%', **kwargs):
         self.style = color_style()
-        super().__init__(*args, **kwargs)
+        if datefmt is None:
+            datefmt = '%d/%b/%Y %H:%M:%S'
+        super().__init__(fmt, datefmt, style, **kwargs)
```

Comment:
I wonder if we can just set `default_time_format` attribute.

Reply:
https://docs.python.org/3.7/library/logging.html#logging.Formatter.formatTime

Reply:
Yes, that's better.

Reply:
Yes, we can use `default_time_format`. but if we use it the log would be like:
`[02/Apr/2020 03:06:23,590] `
With `,590` as milliseconds at the end. this happens in 
https://github.com/python/cpython/blob/5907e61a8d4da6d0f11bf1062d6d17484560a15e/Lib/logging/__init__.py#L582
and that isn't equal with `WSGIRequestHandler` log which doesn't have millisecond at the end:
`[02/Apr/2020 03:06:23] `.

is there any way to change `default_msec_format` to remove `,miliseconds`?

Reply:
What happens if that's the empty string `''`?

Reply:
`TypeError: not all arguments converted during string formatting`
Because of the `s = self.default_msec_format % (t, record.msecs)` needs two places for formating. I think `default_msec_format` should have two places for formating.

Reply:
I think it's fine to make them a bit inconsistent (at least for now). I opened an [issue](https://bugs.python.org/issue40300) in Python.